src/wizard/Step_1/views/Step_1.vue

```diff
@@ -8,4 +68,4 @@ import { Component } from "vue-property-decorator";
 export default class Step_1 extends Vue {}
 </script>
 
```

Comment:
The object below is to be put into an array and passed as a prop to this page from `src\wizard\Step_1\views\Step_1.vue`.

On 'Step_1.vue' the length of this array will determine if the user sees the `CreatePortfolio.vue` component `(if donePortfolio.length ===0)` or the `ViewPortfolio.vue` component `(if donePortfolio.length > 0)`

Either you or Devonte will have to code `src\wizard\Step_1\views\Step_1.vue`.

```
{
  "id": "515e37cb-b7e9-4228-ace8-04cf6fbeb6c3",
  "name": "PORTFOLIO_NAME",
  "description": "PORTFOLIO_DESCRIPTION",
  "csp_provisioning_status": "not_started",
  "dod_component": [
    "air_force",
    "army",
    "marine_corps",
    "navy",
    "space_force",
    "combatant_command",
    "joint_staff",
    "dafa",
    "osd_psas",
    "nsa"
  ],
  "portfolio_managers": [
    "joe.manager@example.com",
    "jane.manager@example.com"
  ],
  "applications": [
    {
      "id": "74f97b58-5d33-4eec-8751-d943828047c7",
      "name": "APPLICATION_NAME",
      "description": "APPLICATION_DESCRIPTION",
      "environments": [
        {
          "id": "765d364f-d7f1-4f28-be8f-123d915b68bb",
          "name": "ENVIRONMENT_NAME",
          "funding_source": "599948c4-35c1-49ee-a7ab-9190ca2c1a81"
        }
      ]
    }
  ]
}
```

Reply:
i understand, but is it CreatePortfolio.vue another ticket?

Reply:
This was address in meeting today.